templates/documentation/datasheet-documentation.html

```diff
@@ -0,0 +1,243 @@
+                                            close
+                                        </span>
+                                        <span>
+                                            ]
+                                        </span>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </li>
+                    <li class="relative border-b border-gray-200">
+                        <button type="button" class="w-full px-8 py-6 text-left"
+                                @click="selected !== 2 ? selected = 2 : selected = null">
+                            <div class="text-2xl flex items-center justify-between">
+                                <span>Step 2. Upload Students CSV </span>
+                                <span class="ico-plus"></span>
+                            </div>
+                        </button>
+                        <div id = "select2" class="relative overflow-hidden transition-all max-h-0 duration-700" style=""
+                             x-ref="container2"
+                             x-bind:style="selected == 2 ? 'max-height: ' + $refs.container2.scrollHeight + 'px' : ''">
+
+                            <div class="text-left items-center">
+                                <div class="p-6">
+                                    <ul class="list-disc pl-8">
+                                        <li class="py-1">
+                                            Admin will be able to upload a preset CSV of students. This will prevent
+                                            admin from having to manually input respective data one by one. Admin will
+                                            see an error message if CSV is in the incorrect format. Follow the
+                                            instructions given in the message and reupload file
+                                        </li>
+                                        <li class="py-1">
+                                            To upload a new file, click [Upload New File]
+                                        </li>
+                                        <li class="py-1">
+                                            Select corresponding category. Click on [Choose File] and select the
+                                            corresponding file to its category
+                                        </li>
+                                    </ul>
+                                    <div class="cursor-default"
+                                         @click="selected !== 2 ? selected = 2 : selected = null">
+                                        <span>
+                                            [
+                                        </span>
+                                        <span class="hover:underline text-blue-500">
+                                            close
```

Comment:
why this weird close button for everything? An icon or something would look much nicer

Reply:
sure will add this